<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMMA Tower Defense</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>AMMA TOWER DEFENSE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24pt; font-weight: bold; color: #ff6600;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="line">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Game Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14pt; font-weight: bold;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mapSizeLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Map Size :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="xmap"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ymap"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="coinLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Starting Coins :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StartCoin"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="settingsButton">
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="helpButton">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_Menu">
    <property name="title">
     <string>Main Menu</string>
    </property>
   </widget>
   <addaction name="menuMain_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>